change/@fluentui-react-conformance-90ba2958-fc3c-4b00-9180-740ecb0fb58e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Not very important but this should technically be patch
```suggestion
  "type": "patch",
```

Reply:
Oops, good catch 👍